webview.h

```diff
@@ -854,6 +856,9 @@ class win32_edge_engine {
   win32_edge_engine(bool debug, void *window) {
     if (window == nullptr) {
       HINSTANCE hInstance = GetModuleHandle(nullptr);
+      if (hInstance == nullptr) {
```

Comment:
I don't think that this check is necessary because `GetModuleHandle` should always return a valid handle when passing in null.